### PR TITLE
+bigstringaf.0.5.3

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/bigstringaf/bigstringaf.0.5.3/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+homepage: "https://github.com/inhabitedtype/bigstringaf"
+bug-reports: "https://github.com/inhabitedtype/bigstringaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/bigstringaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (mseri) [inline packages/bigstringaf/bigstringaf.0.5.3/opam]:
```suggestion
  "dune"
```

Comment (kit-ty-kate) [inline packages/bigstringaf/bigstringaf.0.5.3/opam]:
```suggestion
  "dune" {>= "1.0"}
```
As described in https://github.com/ocaml/opam-repository/pull/14501#issuecomment-515951073

**Comment by seliopou** [comment]
The dune feedback has been addressed.

**Comment by hannesm** [comment]
thanks!

**Pull request merged**